component/toast/toast.css

```diff
@@ -0,0 +1,63 @@
+/* General */
+
+a {
+    text-decoration: none;
+}
+
+/* Typography */
+
+:root {
+    --font-family: system-ui, -apple-system, "Segoe UI", Roboto, "Helvetica Neue",
+    Arial, "Noto Sans", "Liberation Sans", sans-serif, "Apple Color Emoji",
+    "Segoe UI Emoji", "Segoe UI Symbol", "Noto Color Emoji";
+}
+
+/* Toast */
+.toast {
+    width: 400px;
+    position: fixed;
+    display: flex;
+    width: fit-content;
+    background-color: #333333;
+    padding: 10px;
+    border-radius: 5px;
+    align-items: center;
+    gap: 10px;
+}
+
+.top-left {
+    top: 10px;
+    right: 10px;
+}
+
+.bottom-right {
+    bottom: 10px;
+    right: 10px;
+}
+
+.bottom-left {
+    bottom: 10px;
+    left: 10px;
+}
+
+.toast-title {
+    color: hsla(0, 0%, 100%, 0.87);
+    flex-grow: 1;
+    font-size: 1rem;
+}
+
+.toast-action {
+    color: #bb86fc;
```

Comment:
you can define variables for colors.it will be more easy to work with.

Reply:
thanks have made the changes